mmpose/core/evaluation/top_down_eval.py

```diff
@@ -327,35 +422,68 @@ def keypoints_from_heatmaps(heatmaps,
         - maxvals (np.ndarray[N, K, 1]): Scores (confidence) of the keypoints.
     """
 
-    preds, maxvals = _get_max_preds(heatmaps)
     N, K, H, W = heatmaps.shape
-
-    if post_process:
-        if unbiased:  # alleviate biased coordinate
-            assert kernel > 0
-            # apply Gaussian distribution modulation.
-            heatmaps = _gaussian_blur(heatmaps, kernel)
-            heatmaps = np.maximum(heatmaps, 1e-10)
-            heatmaps = np.log(heatmaps)
-            for n in range(N):
-                for k in range(K):
-                    preds[n][k] = _taylor(heatmaps[n][k], preds[n][k])
+    if use_udp:
+        if target_type == 'GaussianHeatMap':
+            coords, maxvals = _get_max_preds(heatmaps)
+            if post_process:
+                coords = post_dark(coords, heatmaps)
+        elif target_type == 'CombinedTarget':
```

Comment:
Please also add a comment to explain briefly, what is "CombinedTarget"  : )

Reply:
i will add the following comment in the args of any functions that involve 'CombinedTarget' ~
        target_type (str): supported targets: 'GaussianHeatMap', 
            'CombinedTarget'. Default:'GaussianHeatMap'
            CombinedTarget: The combination of classification target
            (response map) and regression target (offset map).
            Paper ref: Huang et al. The Devil is in the Details: Delving into 
            Unbiased Data Processing for Human Pose Estimation (CVPR 2020).

Reply:
Nice.